cmd/cmd.go

```diff
@@ -35,6 +40,27 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	dotGauge        = ".gauge"
+	prevCmdFileName = "prevCmd.json"
```

Comment:
`prevCmd.json` filename is a little ambiguous. It can just mean previous gauge command. Can rename it to something else? like `lastRunCmd`?